<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="styleSheet">
       <string notr="true">b</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabMediaCapture">
       <attribute name="icon">
        <iconset resource="../../resource.qrc">
         <normaloff>:/resource/iconCamera.svg</normaloff>:/resource/iconCamera.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Camera</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="mainLayoutMediaCaptureTab">
          <item>
           <layout class="QVBoxLayout" name="vLayoutForCamera">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutCameraVideoSave">
            <property name="spacing">
             <number>7</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <spacer name="lineLeftOfCamera">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="labelRecordingTimer">
              <property name="text">
               <string>00:00:00</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCaptureMedia">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/resource/captureImage.svg</normaloff>:/resource/captureImage.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonNextButtonRecorder">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/resource/nextArrow.svg</normaloff>:/resource/nextArrow.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="lineVidAndSaveDivider">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSaveMediaRec">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/resource/save.svg</normaloff>:/resource/save.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCancelRec">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/resource/cancel.svg</normaloff>:/resource/cancel.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="lineRightOfVideoSave">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMediaPlayer">
       <attribute name="icon">
        <iconset resource="../../resource.qrc">
         <normaloff>:/resource/iconPlayer.svg</normaloff>:/resource/iconPlayer.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>MPlayer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="vLayoutMediaPlayer">
          <item>
           <spacer name="mediPlayerSpacerFill">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="controllerWidgetContainer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="vLayoutMediaPlayerController">
             <item>
              <layout class="QHBoxLayout" name="hLayoutPlayerSliderAndText">
               <item>
                <spacer name="spaceSsliderMediaPlaybackLeft">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="labelMediaElapsedTime">
                 <property name="text">
                  <string>00:00:00</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="sliderMediaPlayback">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelMediaTotalTime">
                 <property name="text">
                  <string>00:00:00</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="spacerSliderMediaPlaybackRight">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="hLayoutMediaControlButtons">
               <item>
                <spacer name="spacerBeforeCodec">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Minimum</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonAudioCodec">
                 <property name="text">
                  <string>Any</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="mediaControllerLeftSpacer3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <spacer name="mediaControllerLeftSpacer4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonFullScreen">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../../resource.qrc">
                   <normaloff>:/resource/fullScreen.svg</normaloff>:/resource/fullScreen.svg</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonMediaRestart">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../../resource.qrc">
                   <normaloff>:/resource/restartMediaPlay.svg</normaloff>:/resource/restartMediaPlay.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="mediaButtonDividerLeft">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonMediaPrev">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../../resource.qrc">
                   <normaloff>:/resource/prev.svg</normaloff>:/resource/prev.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonToggleMedia">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../../resource.qrc">
                   <normaloff>:/resource/playMedia.svg</normaloff>:/resource/playMedia.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonMediaNext">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../../resource.qrc">
                   <normaloff>:/resource/next.svg</normaloff>:/resource/next.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="mediaButtonDividerRight">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonLoadMedia">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../../resource.qrc">
                   <normaloff>:/resource/loadMedia.svg</normaloff>:/resource/loadMedia.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="dividerLoadAndVolume">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonSound">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="../../resource.qrc">
                   <normaloff>:/resource/sound.svg</normaloff>:/resource/sound.svg</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="sliderMediaPlayerVolume">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelMediaVolume">
                 <property name="text">
                  <string>0%</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="mediaControllerLeftSpacer5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxAudioSelector"/>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButtonCaptureMedia</tabstop>
  <tabstop>pushButtonMediaRestart</tabstop>
  <tabstop>pushButtonMediaPrev</tabstop>
  <tabstop>pushButtonToggleMedia</tabstop>
  <tabstop>pushButtonMediaNext</tabstop>
  <tabstop>pushButtonLoadMedia</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
